dev/com.ibm.ws.jbatch.security/src/com/ibm/ws/jbatch/security/impl/WSBatchAuthServiceImpl.java

```diff
@@ -345,6 +509,8 @@ public boolean isInAnyBatchRole() {
     public boolean isAuthorizedInstanceRead(long instanceId)
```

Comment:
Duplicate logic. This should call `authorizedInstanceRead` and catch (and FFDCIgnore) the exception then return true/false.

Reply:
done